src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const tableHeader = document.querySelector('thead');
+const tableBody = document.querySelector('tbody');
+
+function conversion(string) {
+  return string.slice(1).split(',').join('');
+}
+
+tableHeader.addEventListener('click', (event) => {
```

Comment:
Зроби деструктуризацію і змінна heading непотрібна буде
```suggestion
tableHeader.addEventListener('click', ({ target }) => {
```